portfolio/src/main/webapp/script.js

```diff
@@ -73,13 +79,12 @@ function createIndividualComment(comment) {
   deleteButtonElement.setAttribute('class', 'delete-button');
   deleteButtonElement.src = 'images/trash.png';
   deleteButtonElement.addEventListener('click', () => {
-    console.log('here');
     deleteComment(comment);
     // Remove the comment from the DOM.
     liElement.remove();
   });
 
-  usernameElement.innerText = comment.username + ' wrote:';
+  usernameElement.innerText = comment.username + ' wrote on ' + comment.date;
```

Comment:
nit: when you start adding a bunch of '+' in your string concatenation, this is a sign you should probably switch to https://google.github.io/styleguide/jsguide.html#features-strings-template-strings

Same above for `url`.

Reply:
Will do that